extensions/graphite-emitter/README.md

```diff
@@ -0,0 +1,69 @@
+## introduction
+
+This extension emits some selected druid metrics to a graphite carbon server.
+Events are sent after been [pickled](http://graphite.readthedocs.org/en/latest/feeding-carbon.html#the-pickle-protocol) , the size of the batch is configurable.
+Since Graphite doesn't support alerting, alerts will be redirect to warning logs by default. 
+
+ Metrics are organized using the following schema:
+`<namespacePrefix>.[<druid service name>].[<druid hostname>].<druid metrics dimensions>.<druid metrics name>`
+Properly naming your metrics is critical to avoid conflicts, confusing data and potentially wrong interpretation later on.
+
+Example `druid.historical.hist-host1.yahoo.com:8080.MyDataSourceName.GroupBy.query/time`:
+
+ * `druid` -> namespace prefix 
+ * `historical` -> service name 
+ * `hist-host1.yahoo.com:8080` -> druid hostname
+ * `MyDataSourceName` -> dimension value 
+ * `GroupBy` -> dimension value
+ * `query/time` -> metric name
+
+User has control of `<namespace>.[<druid service name>].[<druid hostname>].`
+
+You can omit the hostname by setting `druid.emitter.graphite.ignoreHostname=true`
+`druid.SERVICE_NAME.dataSourceName.queryType.query.time`
+You can omit service name by setting `druid.emitter.graphite.ignoreServiceName=true`
+`druid.HOSTNAME.dataSourceName.queryType.query.time`
+## configuration
+To use graphite as emitter first set `druid.emitter=graphite`
+Also graphite can be used via the composing emitter.
+All the configuration for graphite emitter are under `druid.emitter.graphite`
```

Comment:
This would read better if you said "All the configuration parameters ...".  That would solve the subject-verb number disagreement here.  

It would be good to have some punctuation at the end, maybe a colon.